### PR TITLE
FAQ: Xcode 'file not found' and 'package prefixes'

### LINKED ISSUE #418: Bridging header file not found error
We have successfully translated, compiled and added (with j2objcXcode task) our Java Model to our Xcode Swift project.

However when we import certain classes (does not happen with all) in the bridging header the following error occurs:
```
Bridging-Header.h:6:9: note: in file included from Bridging-Header.h:6:
#import "Template.h"
        ^
Template.h:10:10: error: 'com/test/ReceiverIF.h' file not found
#include "com/test/ReceiverIF.h"
```

Interesting is that the class ReceiverIF was translated correctly and can be found in the generated sources.

### PR DESCRIPTION
**Pull request by brunobowden**
- Explain how to solve ‘file not found’ import errors in Xcode
- Explain how to use Package Prefixes within the plugin
- Fixes #418

### CODE REVIEW

**Comment by brunobowden** [comment]
@ph1lb4, @advayDev1 - please take a look

File: FAQ.md [hunk 1]
```diff
@@ -198,9 +198,10 @@ sourceSets {
 
 ### How do I develop with Swift?
 
-To work with Swift in Xcode, you need to configure a [bridging header](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html#//apple_ref/doc/uid/TP40014216-CH10-XID_81).
-Within that bridging header, include the file needed for using the JRE and any classes that you'd like
-to access from Swift code.
+To work with Swift in Xcode, you need to configure a
+[bridging header](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html#//apple_ref/doc/uid/TP40014216-CH10-XID_81).
+Within that bridging header, include the file needed for using the JRE and any classes that
```
Comment (advayDev1) [inline FAQ.md]: s/file/files

File: FAQ.md [hunk 2]
```diff
@@ -198,9 +198,10 @@ sourceSets {
...
+you'd like to access from Swift code. See note below about import errors.
```
Comment (advayDev1) [inline FAQ.md]: hyperlink

**Comment by advayDev1** [comment]
LGTM with comments + regen the TOC

**Comment by brunobowden** [comment]
How do you update the TOC before publishing it? When I try running it on
the preview page for the PR it didn't work.

File: FAQ.md (hunk 1 above)
Comment (brunobowden) [inline FAQ.md]: done

File: FAQ.md (hunk 2 above)
Comment (brunobowden) [inline FAQ.md]: done

**Comment by advayDev1** [comment]
not on the preview page, go to your branch and view it there, like:
https://github.com/madvay/j2objc-gradle/blob/BRANCH/FAQ.md

**Comment by brunobowden** [comment]
Makes sense...

**Pull request merged**